### PR TITLE
[stable/rabbitmq-ha] Support setting cluster_partition_handling

### LINKED ISSUE #9194: [stable/rabbitmq-ha] Support setting cluster_partition_handling
This is a **FEATURE REQUEST**

**Chart:** stable/rabbitmq-ha

I'd like to be able to configure setting cluster_partition_handling. This is now set to `autoheal`, but it's not he best option for all use cases. So support for a setting a value like called, say, `clusterPartitionHandling` would be welcome.

### PR DESCRIPTION
**Pull request by msvticket**
#### What this PR does / why we need it:

Adding support for setting RabbitMQ automatic partition handling strategy (cluster_partition_handling)

#### Which issue this PR fixes
  - fixes #9194

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.7.8
-version: 1.16.1
+version: 1.17.1
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: 1.17.0

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: stable/rabbitmq-ha/Chart.yaml (hunk 1 above)
Comment (msvticket) [reply stable/rabbitmq-ha/Chart.yaml]: Ah. Fixed the typo.

**Pull request merged**